tools/debugging/stress_test_transfers.py

```diff
@@ -523,6 +524,32 @@ def run_profiler(
     return profiler_processes
 
 
+def get_balance_for_node(node: RunningNode) -> TokenAmount:
+    response = requests.get(f"{node.url}/api/v1/channels").json()
+    if "errors" in response:
+        raise Exception(f"Failed to get channels: {response['errors']}")
+    assert len(response) == 1
```

Comment:
Humm, the mediator will have two channels, this will fail then.

Reply:
It's only called on initiators right now, that's why it works. I can change it to all channels on all nodes.